fe/fe-core/src/main/java/com/starrocks/server/MetadataMgr.java

```diff
@@ -259,6 +261,27 @@ public boolean createTable(CreateTableStmt stmt) throws DdlException {
         }
     }
 
+    public void alterTable(AlterTableStmt stmt) throws UserException {
+        String catalogName = stmt.getCatalogName();
+        Optional<ConnectorMetadata> connectorMetadata = getOptionalMetadata(catalogName);
+
+        if (connectorMetadata.isPresent()) {
+            String dbName = stmt.getDbName();
+            String tableName = stmt.getTableName();
+            if (getDb(catalogName, dbName) == null) {
+                throw new DdlException("Database '" + dbName + "' does not exist in catalog '" + catalogName + "'");
+            }
+
+            if (!listTableNames(catalogName, dbName).contains(tableName)) {
```

Comment:
nit: after we implement the tableExists() interface. you can use it to check.

Reply:
nice

Reply:
you could use tableExists now